docs/essentials/component-basics.md

```diff
@@ -192,7 +192,7 @@ export default {
 </script>
 
 <template>
-  <h4>{{ title }}</h4>
+  <h4>{{ '{{' }} title }}</h4>
```

Comment:
I think this should be `<h4>{{ '{{' }} title {{ '}}' }}</h4>` if I am not wrong

Reply:
Thanks for finding this! Will edit now :+1: